package.json

```diff
@@ -47,7 +47,7 @@
   "dependencies": {
     "@ant-design/icons": "^4.6.2",
     "@mll-lab/js-utils": "^2.0.0",
-    "antd": "^4.16.12",
+    "antd": "^4.18.2",
```

Comment:
Should it be [4.18.7](https://github.com/ant-design/ant-design/releases/tag/4.18.7)?